backend2/src/core/article_parser.py

```diff
@@ -50,6 +51,27 @@ def url(self) -> str:
 
         return f"{self.PMC_BASE_URL}{self.article.external_id}"
 
+    def publish_date(self) -> date:
+        try:
+            year = int(self.tree.findall(".//front/article-meta/pub-date/year")[0].text)
```

Comment:
Is `findall` the right method here? `find` seems more appropriate to me sins we need only one tag not many.

Reply:
There are several date-tags out there